crates/freeze/src/datasets/transactions.rs

```diff
@@ -203,15 +206,17 @@ pub(crate) fn process_transaction(
     store!(schema, columns, input, tx.input.to_vec());
     store!(schema, columns, gas_limit, tx.gas.as_u64());
     store!(schema, columns, success, success);
-    if schema.has_column("n_input_bytes") |
-        schema.has_column("n_input_zero_bytes") |
-        schema.has_column("n_input_nonzero_bytes")
+    if schema.has_column("n_input_bytes")
+        | schema.has_column("n_input_zero_bytes")
+        | schema.has_column("n_input_nonzero_bytes")
+        | schema.has_column("raw")
```

Comment:
move this and line 219 outside of the `if` block. the `store!` macro will perform a check of whether schema contains the `raw column`

Reply:
Thank you. I fixed this.